### ISSUE TITLE
Trade Route has Culture and Science on it and diplomatic Trade Screen not working properly

### ISSUE THREAD

**Issue by darkmark777** [LoneGazebo/Community-Patch-DLL#11221]
_1. Mod Version (X.Y.Z). Current Version: 4.15.2_

4.15.2 43 CIV version with only Community Patch on

_2. DirectX Version_

_3. List of Other Mods_

Community Patch v137 
43 CIVS Community Patch v1
Ethnic Diversity - Steam Version v29 
Faster Aircraft Animation v3 
Historical Religions Compilation v45 
Tectonic Map Script v6

_4. Describe the Issue_

My first trade route went to a city state with 1 science and 1 culture attached to it.
Also, when I'm on the diplomatic screen with a leader and I go to trade, nothing goes into the box. My lux aren't there ready to trade.

______________________________________________________________________________________
_5. Save Game From 1 Turn Before (ALWAYS ATTACH THIS IF POSSIBLE)_

_6. Logs (ALWAYS ATTACH THESE IF POSSIBLE)_

_7. CvMiniDump.dmp File (ATTACH IF REPORTING A GAME CRASH)_

_8. Steps to reproduce the Issue (Optional)_

_9. Screenshots of the Issue (Optional)_

**Comment by bsesar** [comment]
@darkmark777, a savegame, logs, and a screenshot or two would be helpful to debug this issue, potentially even a zipped folder with your mods.

**Comment by axatin** [comment]
I fixed the trade route bug, it was easy enough to reproduce. But I can't reproduce the Trade Screen bug, I need at least a savegame for that. The save files are in C:\Users\...\Documents\My Games\Sid Meier's Civilization 5\ModdedSaves\single, to upload one of the files you need to make it a zip-file first.

Btw science can be given from trade routes also in BNW, only culture can't.

**Comment by darkmark777** [comment]
Yes science can be gained with trade routes to other civs, but this is giving it for city states too, plus the culture.
[AutoSave_0230 AD-1600.zip](https://github.com/user-attachments/files/16615241/AutoSave_0230.AD-1600.zip)
Hope I attached the saved file in zip form correctly for you. 
Yeah, every trade that pops up, I can't see what they want to trade, unless they say they wanted open borders or a defensive pact, then I know, but the 2 boxes are still always empty fyi.

**Comment by axatin** [comment]
This is fixed in the newer versions.
